src/Phinx/Migration/Manager.php

```diff
@@ -134,10 +134,11 @@ public function printStatus($environment, $format = null)
      *
      * @param string    $environment Environment
      * @param \DateTime $dateTime    Date to migrate to
+     * @param boolean   $dryRun      Output SQL rather than executing it
      *
      * @return void
      */
-    public function migrateToDateTime($environment, \DateTime $dateTime)
+    public function migrateToDateTime($environment, \DateTime $dateTime, $dryRun)
```

Comment:
Well, the environment has [internal flags for `adapter` and `wrapper`](https://github.com/robmorgan/phinx/blob/0.4.x-dev/src/Phinx/Migration/Manager/Environment.php#L270-L276), could you use those somehow?